lib/danger/init.rb

```diff
@@ -0,0 +1,23 @@
+module Danger
+  class Init < Danger::Runner
+    self.description = 'Creates a Dangerfile.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.